src/components/billing-chart/billing-chart.tsx

```diff
@@ -52,6 +45,100 @@ interface BillingChartProps {
   diagrammType: "Bar" | "Line";
 }
 
+const TooltipContainer = styled.div`
+  border-radius: 4px;
+  border: none;
+  background: white;
+  padding: 16px;
+`;
+
+const TooltipHeading = styled.h4`
+  font-style: normal;
+  font-weight: 900;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+  color: #202840;
+`;
+
+const TooltipText = styled.p`
+  font-size: 14px;
+  color: #202840;
+`;
+
+const TooltipValue = styled.p`
+  font-weight: bold;
+  font-size: 14px;
+  color: #202840;
+  margin-left: 12px;
+`;
+
+const TooltipRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 8px;
+`;
+
+const TooltipDot = styled.span<{ repositoryColor?: string }>`
+  ${(props: { repositoryColor?: string }) => `color: ${props.repositoryColor}`};
```

Comment:
Props should be used on the value only. This makes it more readable, as you see all css keys in one line.

```
color: `${{ repositoryColor } => repositroyColor }`;
```